sklearn/decomposition/_sparse_pca.py

```diff
@@ -211,7 +312,7 @@ def transform(self, X):
 
         Parameters
         ----------
-        X : array of shape (n_samples, n_features)
```

Comment:
this format with the `of` is now the recommended way. Since you edit a bunch of docstring maybe you can revert these. thx